Cargo.toml

```diff
@@ -9,6 +9,10 @@ readme = "README.md"
 description = "Elliptic curve implementations and wrappers for halo2 library"
 rust-version = "1.63.0"
 
+[[bench]]
+name = "group"
+harness = false
+
```

Comment:
Perhaps we could move this to the bottom along with other `[[bench]]`

Reply:
Fixed in https://github.com/privacy-scaling-explorations/halo2curves/pull/26/commits/94202846e0610c6ae74415949173f1200e3dec2b